pyproject.toml

```diff
@@ -17,39 +17,33 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
-# Required for Python 3.7 for now. See: https://stackoverflow.com/a/73932581/194311
-importlib-metadata = "4.13.0"
-# Used for local development
-nautobot = { version = "^1.4.0", optional = true }
+nautobot = { version = "^2.0.0rc1", optional = true }
+python = ">=3.8,<3.12"
 
 [tool.poetry.dev-dependencies]
+Markdown = "*"
 bandit = "*"
 black = "*"
 coverage = "*"
 django-debug-toolbar = "*"
 # we need to pin flake8 because of package dependencies that cause it to downgrade and
 # therefore cause issues with linting since older versions do not take .flake8 as config
 flake8 = "^3.9.2"
+griffe = "0.30.1"               # Last version with python 3.7 support
```

Comment:
Dropping 3.7 support this and similar can be removed.

Reply:
Removed comments.